odoo/tools/image.py

```diff
@@ -75,7 +75,7 @@ def image_resize_image(base64_source, size=(1024, 1024), encoding='base64', file
 
     if image.size != size:
         image = image_resize_and_sharpen(image, size)
-    if image.mode not in ["1", "L", "P", "RGB", "RGBA"]:
+    if image.mode not in ["1", "L", "P", "RGB"]:
```

Comment:
> Let's just preserve the image mode in image_resize_and_sharpen(), least surprise principle ;-)

Would probably be a good idea *but* you can ask `image_resize_image` to resize to a different file format (e.g. pass in PNG and ask for a JPEG thumbnail) which would still crash. The issue also exists if we try to resize a PNG to GIF or whatever.